From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Tue, 1 Sep 2026 12:34:15 +0200
Subject: test(BUSYBOX): use initramfs.testing as initrd name

The `test_dracut` will specify `$TESTDIR/initramfs.testing` as initrd
name. So use this name in the busybox test case as well.

Origin: upstream, https://github.com/dracut-ng/dracut/commit/4c4db47b1986671ff7a9d03ab8376279a74e8474
---
 test/TEST-15-BUSYBOX/test.sh | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/test/TEST-15-BUSYBOX/test.sh b/test/TEST-15-BUSYBOX/test.sh
index c31cbb7..3ab360d 100755
--- a/test/TEST-15-BUSYBOX/test.sh
+++ b/test/TEST-15-BUSYBOX/test.sh
@@ -36,20 +36,18 @@ test_run() {
         set -x
     fi
     local ret=0
-    local initrd="$TESTDIR/initramfs"
 
     test_dracut \
         --no-hostonly --no-kernel --drivers "" \
-        --modules "base busybox" \
-        "$initrd"
+        --modules "base busybox"
 
     # Check applets the base module would otherwise install from the host.
     # Each must resolve to busybox in the resulting initrd.
-    check_applets_from_busybox "$initrd" ash cp ip ls mv mkdir sleep tr || ret=1
+    check_applets_from_busybox "$TESTDIR/initramfs.testing" ash cp ip ls mv mkdir sleep tr || ret=1
 
     # switch_root must NOT be a busybox symlink as the base module reinstalls
     # the host util-linux version on top of any symlink the busybox module left
-    if lsinitrd "$initrd" | grep -E '^l.* switch_root -> .*busybox'; then
+    if lsinitrd "$TESTDIR/initramfs.testing" | grep -E '^l.* switch_root -> .*busybox'; then
         echo "FAIL: switch_root is a busybox symlink, host version was not preserved" >&2
         ret=1
     fi
